setup.py

```diff
@@ -43,7 +43,7 @@
 
 setup(
     name='gcs-oauth2-boto-plugin',
-    version='1.9',
+    version='1.9.1',
```

Comment:
you shouldn't mess with this file.  just land your fixes and leave release stuff to the maintainers.